lib/class-wp-theme-json-gutenberg.php

```diff
@@ -1003,7 +1004,16 @@ private function get_block_classes( $style_nodes ) {
 			}
 
 			if ( self::ROOT_BLOCK_SELECTOR === $selector ) {
-				$block_rules .= 'body { margin: 0; }';
+				$block_rules .= 'body { margin: 0;';
+
+				$has_font_smoothing_support = _wp_array_get( $this->theme_json, array( 'styles', 'typography', 'fontSmoothing' ) );
```

Comment:
Hey @sagarnasit thanks for the contribution. Technically, this is how we'd want to do it. The only bit I'd think we want to change is to use `settings.typography.fontSmoothing` instead of `styles.typography.fontSmoothing`.